Dangerfile

```diff
@@ -58,16 +66,28 @@ if has_app_changes
   `swift build`
   # Generate master reports
   generate_reports(false, 'master')
+  # Diff and report changes to Danger
   @repos.each do |repo|
-    repo_name = repo.partition('/').last
-    branch = File.read("osscheck/branch_reports/#{repo_name}.txt").split(/\n+/).reject { |c| c.empty? }
-    master = File.read("osscheck/master_reports/#{repo_name}.txt").split(/\n+/).reject { |c| c.empty? }
+    @repo_name = repo.partition('/').last
+    def non_empty_lines(path)
+      File.read(path).split(/\n+/).reject { |c| c.empty? }
+    end
+    branch = non_empty_lines("osscheck/branch_reports/#{@repo_name}.txt")
+    master = non_empty_lines("osscheck/master_reports/#{@repo_name}.txt")
+    @repo = repo
+    def convert_to_link(string)
```

Comment:
This is fragile, but should work for now. An improvement would be to use a more structured reporter, as @marcelofabri and mentioned again in https://github.com/realm/SwiftLint/issues/1025#issuecomment-269105939)